### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by TheDevelolper** [crisbodnar/TensorFlow-NEAT#11]
This is more of a question than an issue. 

Does this approach, in your opinion, increase the likelihood of overfitting. To your train / cross validation set?
